llvm/lib/Target/RISCV/RISCVVLOptimizer.cpp

```diff
@@ -966,6 +966,13 @@ static bool isSupportedInstr(const MachineInstr &MI) {
   case RISCV::VADC_VIM:
   case RISCV::VADC_VVM:
   case RISCV::VADC_VXM:
+  case RISCV::VMADC_VIM:
+  case RISCV::VMADC_VVM:
+  case RISCV::VMADC_VXM:
+  case RISCV::VMSBC_VVM:
+  case RISCV::VMSBC_VXM:
+  case RISCV::VSBC_VVM:
+  case RISCV::VSBC_VXM:
```

Comment:
Nit, match the ordering of the pseudos?

```suggestion
  case RISCV::VSBC_VVM:
  case RISCV::VSBC_VXM:
  case RISCV::VMSBC_VVM:
  case RISCV::VMSBC_VXM:
```